test/scripts/dynamic-fee-teal-test.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+date '+dynamic-fee-teal-test start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+export GOPATH=$(go env GOPATH)
+
+TEMPDIR=$(mktemp -d)
+trap "rm -rf $TEMPDIR" 0
+
+NETDIR=${TEMPDIR}/net
+
+if [ ! -z $BINDIR ]; then
+    export PATH=${BINDIR}:${PATH}
+fi
+
+goal network create -r ${NETDIR} -n tbd -t ${GOPATH}/src/github.com/algorand/go-algorand/test/testdata/nettemplates/TwoNodes50EachFuture.json
+
+goal network start -r ${NETDIR}
+
+# replaces prior trap0
+trap "goal network stop -r ${NETDIR}; rm -rf ${TEMPDIR}" 0
+
+export ALGORAND_DATA=${NETDIR}/Node
+
+ACCOUNT=$(goal account list|awk '{ print $3 }')
+ACCOUNTB=$(goal account new|awk '{ print $6 }')
+ACCOUNTC=$(goal account new|awk '{ print $6 }')
+ACCOUNTD=$(goal account new|awk '{ print $6 }')
+ZERO_ADDRESS=AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ
+LEASE=YmxhaCBibGFoIGxlYXNlIHdoYXRldmVyIGJsYWghISE=
```

Comment:
is that a base64 encoding ? if so, can you also try to add "\" to break it ?

Reply:
done